meson.build

```diff
@@ -132,6 +132,11 @@ if get_option('enable-nnstreamer-backbone')
   add_project_arguments('-DENABLE_NNSTREAMER_BACKBONE=1', language:['c','cpp'])
 endif
 
+tflite_dep = dependency('tensorflow-lite', required: false)
```

Comment:
I wonder if we have to limit the tflite version?

Reply:
I am following nnstreamer here. They support version checks for tf1/2.
I will add them as a separate PR as it will not be that direct for android.